tests/uat/uat_sanity_suite_test.go

```diff
@@ -201,3 +201,17 @@ func TestVirtualDevice2PortCreateConnection_PNFV(t *testing.T) {
 	output := terraform.Output(t, terraformOptions, "port_connection_id")
 	assert.NotNil(t, output)
 }
+
+func TestVirtualDevice2AWSCreateConnection_PFCR(t *testing.T) {
```

Comment:
Similar to the other PR, you have added this test to the PNFV user in GHA, but here it is specified as _PFCR. Please make sure the test is in the right place. If it's actually PFCR then this should be in the PFCR user of GHA.

Reply:
Yes, its PFCR user. Addressed with [this](https://github.com/equinix/terraform-equinix-fabric/pull/121/commits/d7d7aae803c7e9060adc824ea6fd9ad391b36d51) commit.